bin/kano-uixinit

```diff
@@ -18,6 +18,15 @@
 
 . /usr/share/kano-toolset/logging.sh
 
+# If we are in Kano v3, jump to Dashboard Mode, unless instructed otherwise
+if [ `grep -c "3.0.0" /etc/kanux_version` == "1" ] && [ "${DESKTOP_MODE}" != "1" ]; then
```

Comment:
why do we need to check if we are in Kano OS 3.0.0? 

what's `kano-dashboard-supervisor`?

`kano-uixinit` requires more changes and a good clean up : )